### DISCUSSION TITLE
Unhandled device verification error

### DISCUSSION THREAD

**Discussion by red-candle** [community/community#37302] [Other Feature Feedback, Questions, & Ideas]
1. Write a bug report in VS Code.
2. Attempt to "Preview on GitHub". (not logged in)
3. Arrive at the GitHub login page.
4. Enter correct credentials.
5. **"Oops, an unexpected/unknown/unhandled error has occured."** Now what?
6. Attempt to log in separately.
7. Receive a "verify your device" mail … < This is the kind of stuff that makes people write down their passwords, making the whole system less secure.
8. Continue logging in.
9. Write a bug report in GitHub. :(

So it turns out that #5 failed due to the device verification process.

**Comment by ossanna16** [comment]
Hi there @ifyousayso and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
